src/main/features/claimant-response/routes/paid-amount-summary.ts

```diff
@@ -0,0 +1,31 @@
+import * as express from 'express'
+
+import { AbstractPaidAmountSummaryPage } from 'shared/components/ccj/routes/paid-amount-summary'
+import { claimantResponseCCJPath, Paths } from 'features/claimant-response/paths'
+
+import { DraftClaimantResponse } from 'claimant-response/draft/draftClaimantResponse'
+import { AbstractModelAccessor, DefaultModelAccessor } from 'shared/components/model-accessor'
+import { PaidAmount } from 'ccj/form/models/paidAmount'
+
+class PaidAmountSummaryPage extends AbstractPaidAmountSummaryPage<DraftClaimantResponse> {
+
+  getView (): string {
```

Comment:
Please remove as you override default template with the same one. No point doing it.

Reply:
Done